modules/apps/frontend-js/frontend-js-spa-web/test/app/App.ts

```diff
@@ -1296,10 +1451,10 @@ describe('App', function () {
 				return null;
 			}
 		}
-		this.app = new App();
-		this.app.addRoutes(new Route('/path1', NullStateScreen));
-		this.app.navigate('/path1').then(() => {
-			this.app.navigate('/path1#hash').then(() => {
+		app = new App();
+		app.addRoutes(new Route('/path1', NullStateScreen));
+		app.navigate('/path1').then(() => {
+			app?.navigate('/path1#hash').then(() => {
 				window.addEventListener(
 					'popstate',
 					() => {
```

Comment:
`expect(getCurrentBrowserPath(document.referrer))` expects 0 argument but received `document.referrer`. Similar to another comment here, do we just straight up delete `document.referrer` as it "probably" doesn't do anything here?